be/src/olap/rowset/rowset_meta.h

```diff
@@ -233,8 +233,6 @@ class RowsetMeta {
 
     bool delete_flag() const { return _rowset_meta_pb.delete_flag(); }
```

Comment:
this code useless also ?

Reply:
I think, it is useless. Because it is never be set, but collector depend on it and many logic depend on it, I do not know the meaning of the flag.